packages/cas-s3/src/S3Cas.ts

```diff
@@ -98,6 +97,10 @@ export default class S3Cas implements ICasService {
   }
 
   public async read(address: string): Promise<FetchResult> {
+    if (Encoder.isBase64UrlString(address) && address.indexOf('Ei') === 0) {
```

Comment:
do we expect this to be used for ION, ELEM or PHOTON? if no, suggest we remove it.

Reply:
this is for supporting the exiting base64 CID's that have been written in core Index files.

Reply:
I get that, but are those files all just wrong? if they are wrong, let's ship a breaking change and fix it.... they are wrong if they won't ever be used again, and that format won't ever be used in the future... I think this code is preserving a dead branch created by a buggy implementation... I'd rather ship a breaking change now that later.

Reply:
Sure, I'll take that out the base64 compatibility and then update the PR.